src/AutoBuild.jl

```diff
@@ -114,6 +114,10 @@ function build_tarballs(ARGS, src_name, src_version, sources, script,
         deploy_jll = true
         deploy_bin_repo = deploy_repo
         deploy_jll_repo = deploy_repo
+    elseif deploy_bin # make sure bin repo and jll repo match
+        deploy_jll_repo = deploy_bin_repo 
+    elseif deploy_jll 
+        deploy_bin_repo = deploy_jll_repo
```

Comment:
If `deploy` is false, but `deploy_bin` and `deploy_jll` are both true, I think we should instead throw an error if both are not set to the same thing, rather than ignore one of the inputs.  So I would change this to instead say:

```
if deploy
    ....
else if deploy_bin && deploy_jll
    if deploy_bin_repo != deploy_jll_repo
        error("Binaries and JLLs must be deployed to the same repositories")
    end
end
```

Reply:
Thanks for the review, I included those changes in the latest commit!